eva/catalog/catalog_manager.py

```diff
@@ -386,8 +385,6 @@ def create_and_insert_multimedia_table_catalog_entry(
         elif format_type is FileFormatType.IMAGE:
             columns = get_image_table_column_definitions()
             table_type = TableType.IMAGE_DATA
-        else:
```

Comment:
So, removing this will result in `create_and_insert_table_catalog_entry` to fail. But, we won't know why? Is it acceptable or am I missing something?

Reply:
Added `assert`